tuf/util.py

```diff
@@ -898,6 +901,42 @@ def load_json_string(data):
 
 
 
+
+
+
+def load_der_string(data):
+  """
+  <Purpose>
+    Deserialize 'data' (DER string) to a Python object. This supports only
+    what tuf.asn1_codec supports, which at the time of this writing is
+    signed role metadata (timestamp, snapshot, root, or targets) converted into
+    ASN.1 and then encoded as DER.
+
+  <Arguments>
+    data:
+      A DER string, as would be output by e.g.
+      asn1_codec.convert_signed_metadata_to_der()
+
+  <Exceptions>
+    tuf.Error, if 'data' cannot be deserialized to a Python object.
+
+  <Side Effects>
+    None.
+
+  <Returns>
+    Deserialized object.  For example, a dictionary.
```

Comment:
`asn1_codec.convert_signed_der_to_dersigned_json()` returns a tuf.formats.signable_schema object.  Might be clearer to say that instead of just "dictionary".  Consider the name `asn1_codec.convert_signed_der_to_tuf_signable()`, or something along those lines.